test/requires-test.js

```diff
@@ -0,0 +1,515 @@
+var assert = require('assert');
+var bemhtml = require('../').bemhtml;
+var vm = require('vm');
+var EOL = require('os').EOL;
+var vow = require('vow');
+/* jscs:disable */
+// fake commonJS module after browserify
+var FAKE_COMMON_MODULE = 'var require=(function e(t,n,r){function s(o,u){if(!n[o]){if(!t[o]){var a=typeof require=="function"&&require;if(!u&&a)return a(o,!0);if(i)return i(o,!0);var f=new Error("Cannot find module \'"+o+"\'");throw f.code="MODULE_NOT_FOUND",f}var l=n[o]={exports:{}};t[o][0].call(l.exports,function(e){var n=t[o][1][e];return s(n?n:e)},l,l.exports,e,t,n,r)}return n[o].exports}var i=typeof require=="function"&&require;for(var o=0;o<r.length;o++)s(r[o]);return s})({"fake":[function(require,module,exports){ module.exports = { getText: function () { return "Hello templates!"; } }; },{}]},{},[]);';
+var FAKE_COMMON_2MODULES = 'var require=(function e(t,n,r){function s(o,u){if(!n[o]){if(!t[o]){var a=typeof require=="function"&&require;if(!u&&a)return a(o,!0);if(i)return i(o,!0);var f=new Error("Cannot find module \'"+o+"\'");throw f.code="MODULE_NOT_FOUND",f}var l=n[o]={exports:{}};t[o][0].call(l.exports,function(e){var n=t[o][1][e];return s(n?n:e)},l,l.exports,e,t,n,r)}return n[o].exports}var i=typeof require=="function"&&require;for(var o=0;o<r.length;o++)s(r[o]);return s})({"fake":[function(require,module,exports){ module.exports = { getText: function () { return "Hello templates!"; } }; },{}], "fake1":[function(require,module,exports){ module.exports = { getText: function () { return "Good-bye templates...."; } }; },{}]},{},[]);';
+// test texts
+var TEXT = 'Hello templates!';
+var TEXT1 = 'Good-bye templates....';
+/* jscs:enable */
+var chai = require('chai');
+var chaiAsPromised = require('chai-as-promised');
+chai.use(chaiAsPromised);
+
+describe('Require modules', function () {
+  describe('global deps', function () {
+    it('should get dependencies from global scope (Node.js for example)',
+      function () {
+        var code = 'global.text = "' + TEXT + '";';
+        var bundle = bemhtml.generate('', {
+          requires: { textModule: { globals: 'text' } }
+        });
+
+        var sandbox = { global: {}, exports: {} };
+        sandbox.module = { exports: sandbox.exports };
+
+        vm.runInNewContext(code + EOL + bundle, sandbox);
+
+        var result = sandbox.exports.bemhtml.compile(function () {
```

Comment:
Code style: notice that `function() {` in other files in the repo doesn’t have space before `()` brackets.